src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Services/DelegationService.cs

```diff
@@ -215,6 +218,39 @@ public async Task<IEnumerable<Delegation>> CreateClientDelegation(CreateSystemDe
         return await CreateClientDelegations(request.RolePackages, client, facilitator, agentAssignment, options);
     }
 
+    private async Task VerifyDelegationPackages(List<CreateSystemDelegationRolePackageDto> rolepackages)
+    {
+        // Create distinct role+packages dictionary
+        var rolepacks = new Dictionary<string, List<string>>();
```

Comment:
Hvorfor gå via RolePackage i stedet for å bare slå opp package basert på urn? 🤔 

Evt. kunne jo dette ellers blitt håndtert av sjekket at pakken finnes i client assigment, så lenge vi gjør den sjekken før vi oppretter systembruker entitet og agent rolle.

Reply:
Ahh.. rolepackages her er fra input'en. Hode mitt leste dette som at vi leste inn alle RolePackages fra databasen for å så iterere over de 😆

Reply:
Hmm.. innser at siden vi her krever at man oppgir rolle-assignment som klientdelegering av pakken skal knyttes til, så må vi i neste runde kreve at man oppgir "RightHolder" spøkelses-rollen. Men denne vil jo ikke ha noe RolePackage kobling i databasen.